src/test/java/uk/nhs/tis/trainee/notifications/TisTraineeNotificationsApplicationTest.java

```diff
@@ -21,16 +21,29 @@
 
 package uk.nhs.tis.trainee.notifications;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.flyway.FlywayAutoConfiguration.FlywayConfiguration;
+import org.springframework.boot.autoconfigure.quartz.QuartzAutoConfiguration;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
 import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
 @ActiveProfiles("test")
 class TisTraineeNotificationsApplicationTest {
 
+  @Autowired
+  ApplicationContext context;
+
   @Test
   void contextLoads() {
-
+    assertThat("Unexpected flyway bean", context.getBean(FlywayConfiguration.class),
+        notNullValue(FlywayConfiguration.class));
```

Comment:
I must admit I don't know if this is actually a useful test. I need to improve my understanding here.

Reply:
Cool. Removed.